spec/components/works/add_files_component_spec.rb

```diff
@@ -3,12 +3,35 @@
 
 require 'rails_helper'
 
+# rubocop:disable RSpec/VerifiedDoubles
+# rubocop:disable RSpec/AnyInstance
 RSpec.describe Works::AddFilesComponent do
   let(:form) { ActionView::Helpers::FormBuilder.new(nil, nil, controller.view_context, {}) }
   let(:rendered) { render_inline(described_class.new(form: form)) }
 
-  it 'renders the component' do
-    expect(rendered.to_html).to include('Add your files')
-    expect(rendered.css('button.dz-clickable').to_html).to include('Choose files')
+  before do
+    allow_any_instance_of(ActionView::Helpers::FormBuilder).to receive(:object).and_return(object)
```

Comment:
why don't you pass the object on line 9 rather than stubbing?

Reply:
I tried that and it didn't work. NoMethodError: Cannot call `object` on `nil`.

Reply:
The formbuilder instance that is created on line 9 is different than the one instantiated by the view component, apparently. They have different object IDs, which explains the NoMethodError. Hence the gross any_instance jazz.

@jcoyne :point_up: